packages/ds-healthcare-gov/src/components/Header/Header.tsx

```diff
@@ -2,189 +2,65 @@ import ActionMenu from './ActionMenu';
-      return (
-        <div className="ds-u-sm-display--none ds-u-border-bottom--1 ds-u-margin-x--1 ds-u-padding-y--1 hc-c-header__name">
-          {this.props.firstName}
-        </div>
-      );
-    }
-  }
-
-  /**
-   * Event handler for when the "Menu" or "Close" button
-   * within ActionMenu is clicked.
-   */
-  handleMenuToggleClick = () => {
-    this.setState({ openMenu: !this.state.openMenu });
-  };
-
-  /**
-   * Determines which variation of the header should be displayed,
-   * based on the props being passed into the component.
-   * @returns {String} Variation name
-   */
-  variation() {
-    if (this.props.loggedIn) {
-      // Logged-in state, with minimal navigation
-      return VARIATION_NAMES.LOGGED_IN;
-    } else {
-      // Logged-out state, either Learn or Product
-      return VARIATION_NAMES.LOGGED_OUT;
-    }
-  }
-
-  render() {
-    const classes = classnames(
-      `hc-c-header hc-c-header--${this.variation()}`,
-      this.props.className
-    );
-
-    const hasCustomLinks = !!this.props.links;
-    const defaultLinksForVariation = defaultMenuLinks(
-      this.props.initialLanguage,
-      this.props.deConsumer,
-      this.props.subpath,
-      this.props.primaryDomain,
-      this.props.switchLocaleLink,
-      this.props.hideLoginLink,
-      this.props.hideLogoutLink,
-      this.props.hideLanguageSwitch,
-      hasCustomLinks
-    )[this.variation()];
-
-    const links = hasCustomLinks
-      ? this.props.links.concat(defaultLinksForVariation)
-      : defaultLinksForVariation;
-
-    return (
-      <header className={classes} role="banner">
-        <SkipNav href={this.props.skipNavHref} onClick={this.props.onSkipNavClick}>
-          {this.props.t('header.skipNav')}
-        </SkipNav>
-
-        <div className="ds-l-container">
-          <div className="ds-l-row ds-u-align-items--center ds-u-flex-wrap--nowrap ds-u-padding-y--2">
-            <a
-              href={this.props.primaryDomain ? this.props.primaryDomain : '/'}
-              className="hc-c-logo-link ds-l-col ds-l-col--auto"
-            >
-              <Logo locale={this.props.initialLanguage} />
-            </a>
-
-            <ActionMenu
-              firstName={this.props.firstName}
-              onMenuToggleClick={this.handleMenuToggleClick}
-              locale={this.props.initialLanguage}
-              loggedIn={this.props.loggedIn}
-              open={this.state.openMenu}
-              links={links}
-            />
-          </div>
-        </div>
-
-        <Menu
-          beforeLinks={this.beforeMenuLinks()}
-          links={links}
-          open={this.state.openMenu}
-          primaryDomain={this.props.primaryDomain}
-          submenuTop={this.props.submenuTop}
-          submenuBottom={this.props.submenuBottom}
-        />
-
-        {this.props.deConsumer && <DeConsumerMessage deBrokerName={this.props.deBrokerName} />}
-        {this.props.headerBottom}
-      </header>
-    );
-  }
+interface Link {
```

Comment:
[question] would it be helpful to export this interface since it is the type for the `links` prop?

Reply:
Or maybe this is covered by the Header/index.d.ts file?

Reply:
I'll see what I can do